numpy/polynomial/tests/test_hermite.py

```diff
@@ -99,6 +101,16 @@ def test_hermdiv(self):
                 res = herm.hermadd(herm.hermmul(quo, ci), rem)
                 assert_equal(trim(res), trim(tgt), err_msg=msg)
 
+    def test_hermpow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.array(range(i + 1))
+                power = j + 1
```

Comment:
Sorry to keep spotting things - but can you remove the `+1` in both places here? We should be testing that `pow(c, 0)` works too (for all of these cases).

Reply:
Thanks for catching all these things, I appreciate it.